examples/keda-swp-scaling/create-claim.py

```
@@ -0,0 +1,94 @@
+            }
+        }
+    }
+    try:
+        custom_api.create_namespaced_custom_object(
+            group="extensions.agents.x-k8s.io",
+            version="v1beta1",
+            namespace=NAMESPACE,
+            plural="sandboxclaims",
+            body=body
+        )
+    except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)